apis_core/apis_entities/templates/apis_core/apis_entities/abstractentity_detail.html

```diff
@@ -10,6 +10,7 @@
 {% endblock col-zero %}
 
 {% block col-one %}
+  {% include "relations/list_relations_include.html" %}
```

Comment:
do we the possibility to not use the new relations? or do we switch and need to use them as soon as this PR is merged? if the former this needs to be changed as it will break is relations is not in installed apps